<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowTitle">
   <string>Расписание</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QToolButton" name="novayaZadachaKnopka">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Новая задача</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="startStopKnopka">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>5</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Старт/стоп</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="novayaVlozhennayaZadacha">
        <property name="text">
         <string>Новая вложенная задача</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTreeWidget" name="derevoZadach">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>3</verstretch>
       </sizepolicy>
      </property>
      <column>
       <property name="text">
        <string>Имя</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Всего</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Своё</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Максимум</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Минимум</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Название</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="kratkoeOpisanie">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="imyaZadachi">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Описание</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="minimumFlag">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Минимум</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="maximumFlag">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Максимум</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QTimeEdit" name="maximumPeriod">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QTimeEdit" name="minimumPeriod">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="detalnoeOpisanie">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>403</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Задача</string>
    </property>
    <addaction name="novayaZadachaMenu"/>
    <addaction name="novayaVlozhennayaZadachaMenu"/>
    <addaction name="udalitZadachuMenu"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="novayaZadachaMenu">
   <property name="text">
    <string>Создать</string>
   </property>
  </action>
  <action name="novayaVlozhennayaZadachaMenu">
   <property name="text">
    <string>Создать вложенную</string>
   </property>
  </action>
  <action name="udalitZadachuMenu">
   <property name="text">
    <string>Удалить</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
